docs/docs/en/guide/task/pytorch.md

```diff
@@ -0,0 +1,117 @@
+# Pytorch Node (experimental)
+
+## Overview
+
+[Pytorch](https://pytorch.org) is a mainstream Python machine learning library.
+
+In order to enable users to **run Pytorch project in DolphinScheduler more convenient,** Pytorch task plugin is implemented. It provides convenient Python environment management and supports the running of Python projects.
```

Comment:
```suggestion
`Pytorch` task plugin enables users to run `Pytorch` projects in DolphinScheduler more conveniently. In addition, it supports handy Python environment management.
```

Reply:
Please avoid using passive voice unless necessary in tech doc writing.

Reply:
I see. Thank you. I'll be careful in the future.